Assets/VRTK/Source/SDK/Base/SDK_DescriptionAttribute.cs

```diff
@@ -138,10 +141,17 @@ public SDK_DescriptionAttribute(Type typeToCopyExistingDescriptionFrom, int inde
 
         public static SDK_DescriptionAttribute[] GetDescriptions(Type type)
         {
+#if UNITY_WSA && !UNITY_EDITOR
+            return type.GetTypeInfo().GetCustomAttributes(typeof(SDK_DescriptionAttribute), false)
```

Comment:
I'd prefer any `#if-#else-#endif` to only surround the one line that changed to prevent duplicated code getting out of sync between the two branches.

Here only the first line seems to be different.